wca-guidelines.md

```diff
@@ -49,9 +49,10 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-2><competitors><competitors> Article 2: Competitors
 
 - 2c+) [ADDITION] First-time competitors should register using their legal name. They may register using a reasonable nickname, at the discretion of the WCA Delegate.
-- 2c++) [ADDITION] Competitors must not provide intentionally misleading information, and returning competitors should provide information consistent with past information (e.g. exact name and WCA ID).
 - 2c1+) [ADDITION] The requirements for a completed registration must be clearly stated on the competition website.
 - 2c1++) [EXAMPLE] For example, the organization team may require a confirmation e-mail to be answered before a particular date, pre-payment of the registration fee, or paying due registration fees of past competitions. The organization team should carefully consider the fairness of the requirements.
+- 2c2+) [CLARIFICATION] A WCA profile is the collection of data related to a competitor, and includes their personal information (name, nationality, gender and date of birth) as well as all their results in official WCA competitions.
+- 2c2++) [CLARIFICATION/ADDITION] If a competitor wishes to change or update their personal information between competitions (e.g. a change of nationality, name or gender), they must [contact the WCA Results Team](mailto:results@worldcubeassociation.org).
```

Comment:
ADDITION, I think. I'd even put this in the Regs.

Could you change the email to a WCA website link? Whatever we put here will be hardcoded into future historical revisions (inc. PDFs).

Reply:
I don't think there's a specific website for this. We have https://www.worldcubeassociation.org/contact/dob, which is only for date of birth fixing.

Second best option is https://www.worldcubeassociation.org/contact/website, which I also find too generic.

Reply:
Yeah, I'd also like there to be a better link before merging. Could you work on making one?

Reply:
I'll ask the team.

Reply:
How about just using https://www.worldcubeassociation.org/contact?

I don't think that I want a special contact form for the WRT. Instead of spending energy building one, we could rather transform the current DOB correction form into a slightly more general persond ata correction form.